wcivf/apps/elections/views/postcode_view.py

```diff
@@ -68,7 +68,7 @@ def get_context_data(self, **kwargs):
             ballot_dict = self.get_ballot_dict()
             context["address_picker"] = ballot_dict.get("address_picker")
             context["addresses"] = ballot_dict.get("addresses")
-        except (InvalidPostcodeError, DevsDCAPIException) as exception:
+        except (InvalidPostcodeError, InvalidUprnError) as exception:
             raise exception
```

Comment:
```py
...
except Exception as e:
    raise e
```

is basically a no-op. Catching an error and re-throwing it (with no other action) is the same as just not catching it
I think you can just completely remove the try/catch round this block